data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/RequestHandler.java

```diff
@@ -91,4 +111,16 @@ public void handle(final HttpServerRequest request) {
     producer.send(record, promise);
     return promise.future();
   }
+
+  @Override
+  public Future<Void> reconcile(Map<Broker, Set<Trigger<CloudEvent>>> objects) {
+
+    final var brokers = objects.keySet().stream()
+        .map(b -> "/" + b.namespace() + "/" + b.name())
```

Comment:
Any utils method to format the broker path? (You don't need to do it in this pr)

Reply:
I see this cloned some times and it might be useful to have it